frontend/src/views/shop/detail/DetailSitespeed.vue

```diff
@@ -62,4 +74,181 @@ function formatBytes(bytes: number, decimals = 2): string {
     const i = Math.floor(Math.log(bytes) / Math.log(k));
     return parseFloat((bytes / Math.pow(k, i)).toFixed(dm)) + ' ' + sizes[i];
 }
+
+Chart.register(...registerables);
+
+const timeChartCanvas = ref<HTMLCanvasElement | null>(null);
+const timeChartInstance = ref<Chart | null>(null);
+const transferSizeChartCanvas = ref<HTMLCanvasElement | null>(null);
+const transferSizeChartInstance = ref<Chart | null>(null);
+const clsChartCanvas = ref<HTMLCanvasElement | null>(null);
+const clsChartInstance = ref<Chart | null>(null);
+
+const timeMetrics = [
+  { key: 'ttfb', label: 'TTFB' },
```

Comment:
Please use 4 spaces as in the rest of the file.

Reply:
`make lint-fix` does not change the files.

I adjusted the file.